### PR TITLE
Wikidata records

### PR DESCRIPTION
**Pull request by sumirp**
Updated Wikidata database annotations

### CODE REVIEW

**Comment by bgyori** [comment]
Hi @sumirp, thank for these! It looks like the changes conflict with the latest state of bioregistry.json. Would you be able to resolve these? Let me know if I can help.

File: src/bioregistry/data/bioregistry.json [hunk 1]
```diff
@@ -3482,6 +3808,10 @@
         "omics": "03117"
       }
     },
+    "wikidata": {
+      "database": "Q4783563"
+    },
+    "references" : [""],
```
Comment (bgyori) [inline src/bioregistry/data/bioregistry.json:3814]: Should this be the link to the PR?

File: src/bioregistry/data/bioregistry.json [hunk 2]
```diff
@@ -3685,6 +4023,10 @@
       "miriam": "ardb",
       "n2t": "ardb"
     },
+    "wikidata": {
+      "database": "Q112205944"
+    },
+    "references": [""],
```
Comment (bgyori) [inline src/bioregistry/data/bioregistry.json:4029]: Same as above, this should probably be a link to this PR.

**Comment by sumirp** [comment]
@bgyori Hi, I am updating the entries with the link to the PR.

**Comment by sumirp** [comment]
Hi @bgyori I have made the changes.

**Comment by bgyori** [comment]
Thanks @sumirp but the branch still has conflicts with bioregistry.json on main (due to the fact that you started this branch a few months ago and it has since evolved). Do you know how to resolve these conflicts or should I help do that?

**Comment by sumirp** [comment]
@bgyori Thank you. Could please share what was the issue?

**Comment by bgyori** [comment]
Tests are passing now, the changes I made involved:
- Fixing merge conflicts with respect to the latest main
- Moving a wikidata entry to the appropriate level
- Running linting to fix a number of JSON formatting issues

**Pull request merged**